<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEdit</class>
 <widget class="QWidget" name="UserEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="NorRBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>教师</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="InerLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="ManRBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="NewPassLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>新建用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancleBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccountLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>账号类型：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPassEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SurePassEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SurePassLab">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>270</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="stuRBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>学生</string>
   </property>
  </widget>
  <zorder>TitleLab</zorder>
  <zorder>InerLab</zorder>
  <zorder>CloseBtn</zorder>
  <zorder>NorRBtn</zorder>
  <zorder>PasswordLab</zorder>
  <zorder>ManRBtn</zorder>
  <zorder>PasswordEdit</zorder>
  <zorder>NewPassLab</zorder>
  <zorder>label</zorder>
  <zorder>CancleBtn</zorder>
  <zorder>UserLab</zorder>
  <zorder>AccountLab</zorder>
  <zorder>UserLineEdit</zorder>
  <zorder>SaveBtn</zorder>
  <zorder>NewPassEdit</zorder>
  <zorder>SurePassEdit</zorder>
  <zorder>SurePassLab</zorder>
  <zorder>stuRBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
